src/core/geometry/junction/Junction.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <string>
+#include "core/util/Identification.h"
+#include "core/geometry/coordinate/Coordinate.h"
+#include "core/util/Class.h"
+
+namespace semba::geometry::junction {
+	typedef geometry::CoordId CoordIdForJunctions;
+	class Junction {
+
+	public:
+		static const std::string INVALID_ATTRIBUTE_MESSAGE;
+
+		Junction(std::string & name);
+
+		std::string const& getName() const;
+		std::string getName();
```

Comment:
Same value return comment as for Bundle.

Reply:
Addressed in commit e0876b681447e7be6b8220111352cf1fac6c7c78.